common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/packets/WorldPackets.java

```diff
@@ -535,6 +537,32 @@ else if (particleId == 36)
                 });
             }
         });
+
+        // Incoming Packets
+        protocol.registerServerbound(ServerboundPackets1_13.PLAYER_BLOCK_PLACEMENT, wrapper -> {
+            Position pos;
+            wrapper.write(Type.POSITION, pos = wrapper.read(Type.POSITION)); // Location
```

Comment:
Consider using map() instead of write(read()) like done in the SPAWN_PARTICLE handler

Reply:
Since all you need here is the position, you can remove the wrapper calls below this and change it to
`Position pos = wrapper.passthrough(Type.POSITION)`